WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.java

```diff
@@ -4,12 +4,20 @@
 
 import androidx.appcompat.widget.Toolbar;
 import androidx.test.espresso.ViewInteraction;
+import androidx.test.espresso.matcher.ViewMatchers;
 
 import org.wordpress.android.R;
 
 import static androidx.test.espresso.Espresso.onView;
 import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.hasSibling;
 import static androidx.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.matcher.ViewMatchers.isDescendantOfA;
```

Comment:
Seems to be a lot of unused imports added here, can we remove any we're not using please.